src/Docnet.Core/DocLib.cs

```diff
@@ -131,6 +131,25 @@ public byte[] Split(byte[] bytes, int pageFromIndex, int pageToIndex)
             return _editor.Split(bytes, pageFromIndex, pageToIndex);
         }
 
+        /// <inheritdoc />
+        public byte[] Split(string filePath, string pageRange)
+        {
+            Validator.CheckFilePathNotNull(filePath, nameof(filePath));
+            Validator.CheckPageRangeNotNullOrEmpty(pageRange, nameof(pageRange));
+
+            return _editor.Split(filePath, pageRange);
+        }
+
+        /// <inheritdoc />
+        public byte[] Split(byte[] bytes, string pageRange)
+        {
+            Validator.CheckBytesNullOrZero(bytes, nameof(bytes));
+            Validator.CheckPageRangeNotNullOrEmpty(pageRange, nameof(pageRange));

```

Comment:
Can we validate that page range is correct properly, i.e. if we support `[n,m-o,k]` type, check for `[`, split on `,` and see that each range is fine, because otherwise we are passing this string directly to unmanaged code and it might blow up in an unexpected way

Reply:
Could you take a look again, please? A bit refactored it